lib/request-handler.js

```diff
@@ -251,7 +251,7 @@ RequestHandler.prototype.prepareMultiple = function (queries, callbacksArray, op
   Error.captureStackTrace(this.stackContainer = {});
   var self = this;
   this.retryHandler = function () {
-    //Use a custom retryHandler
+    // Use a custom retryHandler
     self.prepareMultiple(queries, callbacksArray, options, callback);
```

Comment:
Not related to the change, but does this retry preparing indefinitely?  Any concerns about this running repeatedly?  Although looking at the cases it would be exercised (socket error, overloaded/is_bootstrapping/truncate, operation timeout) it doesn't seem like it would be too big of a concern except for maybe the timeout case for whatever reason (i.e. if the prepare behavior causes some weird pause/delay on the cassandra side every time).